docs/pepembed/README.md

```diff
@@ -0,0 +1,102 @@
+
+**4. Insert Embeddings:**  
+Finally, we insert the embeddings into a [qdrant](https://qdrant.tech/) instance. qdrant is a **vector database** that is designed to store embeddings as first-class data types as well as supporting native graph-based indexing of these embeddings. The allows for near-instant search and retrieval of nearest embeddings neighbors given a new embedding (say an encoded search query on a web application). qdrant supports arming the embeddings with a [**payload**](https://qdrant.tech/documentation/payload/) where we store basic information on that PEP like registry path, row id, and its description.
+
+## Install and Run
+While simple to install and run, `pepembed` requires lots of information to function. There are three key aspects: 1) The pephub instance, 2) the qdrant instance, and 3) the keywords. Ensure the following before running the `cli`:
+
+### Setup
+**1. PEPhub instance:**  
+Make sure you have access to a running pephub instance store with peps. Once complete, you can use the following environment variables to tell `pepembed` where to get data. Alternatively, you can pass these as command-line args:  
+* `POSTGRES_HOST`
+* `POSTGRES_DB`
+* `POSTGRES_USER`
+* `POSTGRES_PASSWORD`
+
+**2. Qdrant instance:**  
+In addition to a pephub instance, you will need a running instance of qdrant. It is quite simple and instructions can be found [here](https://qdrant.tech/documentation/quick_start/). The TL;DR is:  
+
+```console
+docker pull qdrant/qdrant
+docker run -p 6333:6333 \
+    -v $(pwd)/qdrant_storage:/qdrant/storage \
+    qdrant/qdrant
+```
+
+This will give you a qdrant instance served at http://localhost:6333. You can pass this information to `pepembed` as environment variables. Alternatively, you may pass these as command-line args:  
+* `QDRANT_HOST`
+* `QDRANT_PORT`
+* `QDRANT_API_KEY`
+* `QDRANT_COLLECTION_NAME`
+
+*Unless you are running this for production, you most likely do not need to specify any of these.*
+
+**3. Keywords:**  
+Finally, we need a keywords file. This is technically optional, and `pepembed` comes with [default keywords](pepembed/const.py), but you may supply your own as a plain text file. This can be supplied only as command-line args:
+* `KEYWORDS_FILE`
+
+There are many other options as well (like specifying the transformer model to use), but the defaults work great for a first try. Use `pepembed --help` to see all options. If you are like me, and like to keep your secrets in a `.env` file, you can export them easily to the environment with `export $(cat .env | xargs)`
+
+### Install
+
+Clone this repository and install with `pip`:
+
+```console
+pip install .
+```
+
+### Run
+
+```console
+pepembed \
+  --keywords-file keywords.txt \
+  --postgres-host $POSTGRES_HOST \
+  --postgres-user $POSTGRES_USER \
+  --postgres-password $POSTGRES_PASSWORD \
+  --postgres-db $POSTGRES_DB \
+```
```

Comment:
what is `pepquery` referenced in the image?

Reply:
It _was_ an imagined microservice that converts text to embeddings...

But now that's just pephub. So I can change that

Reply:
That image is a `png` for some reason, so I need to change it to SVG and edit it to change that